### DISCUSSION TITLE
ListPart pager issue : how to use full pager?

### DISCUSSION THREAD

**Discussion by thanhpt168** [OrchardCMS/OrchardCore#7092] [Q&A]
Hi all,
I am using OrchardCore to build my site. When render ListPart by liquid, it render pagination by using PagerSlim shape by default. It just show: Back Next.
I neet to render my pager like this:  Back 1 2 3 4 5 Next
Is there any way to do that with a custom liquid template?
I googled much but no luck!
Thank you very much.

**Comment by Skrypt** [comment]
Need to check but I think by default the ListPart returns a PagerSlim with it's shape. So, right now I'd say that the option is not available and it would require some development to make it use a full Pager as far as I can remember.

**Reply by thanhpt168** [reply]
Oh, I hope it will be developed soon. Thank you very much!

**Reply by hamzayilmaz** [reply]
Same issue for me, @Skrypt 
Will ever be developed?
What is the best approach to work around this?

**Reply by ns8482e** [reply]
I guess you should be able to override PagerSlim

**Comment by adrianyadav** [comment]
Anyone got an answer for this?

I also want to show blog posts using the full pager instead of just previous and next

**Reply by sultanymbae** [reply]
> Anyone got an answer for this?
> 
> I also want to show blog posts using the full pager instead of just previous and next

Hello, did you find the way how to add full pager instead of just previous and next?
